### ISSUE TITLE
Fall-through to default switch case duplicates the switch case consequent code block

### ISSUE THREAD

**Issue by dbrattli** [fable-compiler/Fable#2357]
### Description

Last fall-through to default switch case duplicates the switch case consequent code block. The generated code is correct so this is not a bug, but the cases should ideally be collapsed so the duplicated code is removed.

### Repro code

```fs
module Test.Test

let x = 10
match x with
| 1
| 2
| 3
| 4
| _ ->
    printfn "Default"
    printfn "Gets"
    printfn "Duplicated"
```
### Expected and actual results

Actual result:

```js
import { printf, toConsole } from "./.fable/fable-library.3.1.1/String.js";

export const x = 10;

switch (x) {
    case 1:
    case 2:
    case 3:
    case 4: {
        toConsole(printf("Default"));
        toConsole(printf("Gets"));
        toConsole(printf("Duplicated"));
        break;
    }
    default: {
        toConsole(printf("Default"));
        toConsole(printf("Gets"));
        toConsole(printf("Duplicated"));
    }
}
```
### Related information

* Fable version: 3.1.1
* Operating system: Mac OS X 11.1

**Comment by alfonsogarciacaro** [comment]
Thanks for reporting! I've sent a fix in #2358. I hope it doesn't break the build :)

Your example just becomes `switch (x) { default: ... }` so we could remove the switch altogether. But I assume these cases are going to be rare (users wouldn't use pattern matching if there are no alternatives) that we don't need to worry.

**Comment by dbrattli** [comment]
Thanks for fixing!!
